src/components/form/examples/form-span-fields.tsx

```diff
@@ -0,0 +1,215 @@
+ * Now if the container's width decreases, it will forces the form to render its
+ * layout in 3 columns instead. Therefore, the second field has to jump
+ * to the next line to still be able to stretch through 3 columns.
+ * This will leave 2 empty cells in the first row, right after the first field.
+ *
+ * To avoid having these empty cells in the UI, limel-form will automatically
+ * bring up two other fields that span only 1 column, and places them in the
+ * available empty cells in the first row.
+ *
+ * However, you can disable this functionality by setting `dense` to fall in the
+ * options for the grid layout
+ *
+ * ```ts
+ * export const schema = {
+ *     type: 'object',
+ *     lime: {
+ *         layout: {
+ *             type: 'grid',
+ *             dense: false,
+ *         },
+ *     },
+ * };
+ * ```
+ *
+ * :::note
+ * Sometimes order of fields are important in the way users perceive the form.
+ * If you choose to have the default auto-reordering behavior of the forms,
+ * make sure to test yout forms layout in different screen sizes to see whether
+ * you can mitigate unwanted layout changes.
+ *
+ * You can avoid unwanted results by changing order of fields so that they render
+ * appropriately on different screens, or by dividing the form into more
+ * sections.
+ * :::
+ *
+ * ###### Stretching a field vertically
+ * All standard elements that can be used in forms such as `limel-input`,
+ * `limel-select`, `limel-slider`, etc. have a fixed UI in the horizontal axis
+ * and therefore it does not really make sense to stretch them in height,
+ * and we strongly recommend you not to!
+ *
+ * But there are some exceptions. One of them is the `textarea`.
+ * It's always recommended to assign at least two rows for a textarea, to improve
+ * its usability. Since this is a component that is supposed to show multi-lined
+ * pieces of text.
+ *
+ * Also, if you make create a custom component for your form –let's say a map–,
+ * you will have the possibility of using `rowSpan`, to increase the height of
+ * your custom component.
+ * ````ts
+ * export const schema = {
+ *     type: 'object',
+ *     properties: {
+ *         comment: {
+ *             type: 'string',
+ *             title: 'Comment',
+ *             lime: {
+ *                 component: {
+ *                     props: {
+ *                         type: 'textarea',
+ *                     },
+ *                 },
+ *                 layout: {
+ *                       colSpan: 3,
+ *                       rowSpan: 2,
+ *                 },
+ *             },
+ *         },
+ *     },
+ * };
+ * ````
+ *
+ * :::warning
+ * Custom web-components that you include in the forms should not have hard-coded
+ * `width` or `height` values! Otherwise they will stretch out of their cell and break
+ * the UI. Make sure that such components are internally designed to be responsive,
+ * and that their `:host` and their eventual wrapping container has the following
+ * styles:
```

Comment:
```suggestion
 * Custom web-components that you include in the forms should not have hard-coded
 * `width` or `height` values! Otherwise they will stretch out of their cell and break
 * the UI. Make sure that such components are internally designed to be responsive,
 * and that their `:host` and any potential wrapping container has the following styles:
```

In English, "eventual" means something that is certain to happen, but at some uncertain time in the future.